app/controllers/granules_controller.rb

```diff
@@ -111,4 +111,29 @@ def replace
       redirect_to collection_path(id: 1, record_id: collection.records.first.id)
     end
   end
+
+  def ingest_specific
```

Comment:
if the controller action and the model method have the same name, could that create confusion down the road?

Reply:
I don't think so, but I'm up for suggestions to rename it?